src/custom_modules/DAO_modules/donations.ts

```diff
@@ -17,6 +19,8 @@ var DAO;
  * @prop {number} transactionCost
  * @prop {Date} timestamp Timestamp of when the donation was recieved
  * @prop {string} method The name of the payment method used for the donation
+ * @prop {string} paymentExternalRef The reference in the external payment handlers' systems
+ * @prop {string} metaOwnerID Who owns the data (EA Norge, Effekt, or joint ownership)
  * @prop {string} KID
```

Comment:
@herleik 

Have you double checked that this function is not being used elsewhere where this might cause an issue? :) A shift + ctrl + f on the function name should reveal this quickly.

Reply:
I had not! But looks fine (assuming you meant the fact that getByID returns an object with more fields than before). Every use just seems to access the fields they need with the dot operator, no iteration.
In effekt-admin, only donation.tsx dispatches the action that ends up calling this.